test/unit/extract/xbrl_test.py

```diff
@@ -180,40 +185,27 @@ def test_xbrl2sqlite_db_exists_yes_clobber(mocker, tmp_path):
     mocker.patch("pudl.extract.xbrl.FercXbrlDatastore", return_value=mock_datastore)
 
     # always use tmp path here so that we don't clobber the live DB when --live-dbs is passed
-    ferc1_sqlite_path = tmp_path / "ferc1_xbrl.sqlite"
+    ferc1_sqlite_path = PudlPaths().output_dir / "ferc1_xbrl.sqlite"
     ferc1_sqlite_path.touch()
-
-    # mock the db path so we can assert it gets clobbered
-    mock_db_path = mocker.MagicMock(spec=ferc1_sqlite_path)
-    mock_pudl_paths = mocker.MagicMock(
-        spec=PudlPaths(), sqlite_db_path=lambda _x: mock_db_path
-    )
-    mocker.patch("pudl.extract.xbrl.PudlPaths", return_value=mock_pudl_paths)
-
     settings = FercToSqliteSettings(
-        ferc1_dbf_to_sqlite_settings=Ferc1DbfToSqliteSettings(),
         ferc1_xbrl_to_sqlite_settings=Ferc1XbrlToSqliteSettings(),
-        ferc2_xbrl_to_sqlite_settings=None,
-        ferc6_xbrl_to_sqlite_settings=None,
-        ferc60_xbrl_to_sqlite_settings=None,
-        ferc714_xbrl_to_sqlite_settings=None,
     )
 
     context = build_op_context(
         resources={
             "ferc_to_sqlite_settings": settings,
             "datastore": "datastore",
-        },
-        config={
-            "workers": 10,
-            "batch_size": 20,
-            "clobber": True,
+            "runtime_settings": RuntimeSettings(
+                clobber=True,
+                xbrl_batch_size=20,
+                xbrl_num_workers=10,
+            ),
         },
     )
 
-    xbrl2sqlite(context)
-
-    mock_db_path.unlink.assert_any_call()
+    assert ferc1_sqlite_path.exists()
+    xbrl2sqlite_op_factory(XbrlFormNumber.FORM1)(context)
+    assert not ferc1_sqlite_path.exists()
```

Comment:
Will this clobber the live SQLite DB if run with `--live-dbs`?

Reply:
This method bails out early if `--live-dbs` is enabled. 

In the long run, we should ditch this flag and assume `--live-dbs` always, and ensure that we're not mucking with `PUDL_OUTPUT` in the tests at all.

I'm not aware if there's anything we could use to force read-only access on the PUDL_OUTPUT directory, so until then, we just have to rely on a good unit/integration test hygiene.

Reply:
Ah, right, OK. Yeah, we should operate within the confines of our current `--live-dbs` world but scheme to overthrow the system and build a better world in the future.